paper/paper.md

```diff
@@ -0,0 +1,122 @@
+bibliography: paper.bib
+---
+
+
+# Summary
+
+A major advantage of magnetic resonance imaging (MRI) over other neuroimaging
+methods is its capability to noninvasively locate a region of interest (ROI) in
+the human brain. For example, using functional MRI, we are able to pinpoint
+where in the brain a certain cognitive process shows higher activation than
+usual. But just knowing the Cartesian coordinate of such a ROI is not useful if
+we cannot connect it with a particular brain region. For this reason, MRI images
+are usually normalized into a common template space [@fonov2011unbiased], where
+well-established atlases can be used to associate a given coordinate with the
+label of a brain region. Most major neuroimaging software packages provide some
+functionality to locate the main peaks of an ROI but are most often restricted
+to a few atlases, often requires manual intervention, do not give the user much
+flexibility in the output creation process, and never considers the full extent
+of the ROI.
+
+To tackle those shortcomings, we created AtlasReader, a Python interface for
+generating coordinate tables and region labels from statistical MRI images. With
+AtlasReader, users can use any of the freely and publicly available neuroimaging
+atlases, without any restriction to their preferred software package, to create
+publication-ready output figures and tables, that contain relevant information
+about the peaks and clusters extent of each ROI. To our knowledge, providing
+atlas information about the full extent of a cluster (i.e. the extent of overlap
+between the ROI and atlas regions) is a new feature that, so far, is not
+available in any other comparable neuroimaging software package.
+
+Executing AtlasReader on an MRI image will lead to the creation of the following
+four kinds of outputs:
+
+1. An **overview figure** showing the ROIs throughout the whole brain at once
+   (see Figure 1).
+2. For each ROI, an **informative figure** showing the sagittal, coronal and
+   transversal plane centered on the main peak of the ROI (see Figure 2).
+3. A **table** containing relevant information about the main **peaks** in each
+   ROI (see Figure 3).
+4. A **table** containing relevant information about the **cluster extent** of
+   each ROI (see Figure 4).
+
+Users have many parameters available to guide the creation of these outputs. For
+example, with `cluster_extent` a user can specify the minimum number of
+contiguous voxels required for a ROI to be shown in the output, `min_dstinace`
+can be used to extract peak information of more than just one peak within a
+given ROI, and `atlas` can be used to specify which atlases should be used for
+the output creation. By default, AtlasReader uses the AAL, the Desikan-Killiany,
+and the Harvard-Oxford atlases (see Figure 5). In the current version, users
+have also access to the Aicha, the Destrieux, the Juelich, the Marsatlas, the
+Neuromorphometrics, and the Talairach atlas.
```

Comment:
Should these atlases all have citations? Or perhaps we could clarify that the citations are included in the AtlasReader distribution.

Reply:
The problem is, some atlases have multiple references and MarsAtlas and Neuromorphics have none (see [here](https://github.com/miykael/atlasreader/tree/master/atlasreader/data#marsatlas)). Should we nonetheless include them?

Reply:
```suggestion
Neuromorphometrics, and the Talairach atlas. Further details about the individual atlases,
how to acknowledge them, and their license requirements are detailed in the
[atlasreader/data](https://github.com/miykael/atlasreader/tree/master/atlasreader/data) directory.
```

I don't think there's an easy answer to this discussion so I've suggested a link. I think this is something that the reviewers will either realise is difficult and therefore we've done our best, or they'll have a request and we can do that 😸